build/Uno.WinUI.nuspec

```diff
@@ -894,6 +894,21 @@
 	<file src="Uno.WinUI.targets" target="buildTransitive\net7.0-macos" />
 	<file src="Uno.WinUI.targets" target="buildTransitive\net5.0-windows" />
 
+	<file src="Uno.WinUI.props" target="buildTransitive\MonoAndroid" />
```

Comment:
The only thing this props file does is:

https://github.com/unoplatform/uno/blob/d077eaf8b9337c55c0fd9e912e65f99088a3ab9d/build/uno.winui.common.props#L4-L10

1. `UnoForceSingleTFM` doesn't seem to be set to true anywhere, so this effectively does nothing
2. There is no `uno.vscode.targets`

I think we can just delete the props file as dead.

Reply:
This file is now required by the vscode extension, both the property and the props file are defined there as well. (the name was incorrect, I fixed it).